pom.xml

```diff
@@ -3765,9 +3777,9 @@
         <scope>test</scope>
       </dependency>
       <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.12</version>
+        <groupId>org.opennms.core.test-api</groupId>
+        <artifactId>org.opennms.core.test-api.junit</artifactId>
+        <version>4.12</version> <!-- if you change this, be sure to update core/test-api/junit/pom.xml as well! -->
```

Comment:
Can't we just define it as a property, e.g. `${junitVersion}` so we don't have to worry about the comment?

Reply:
Because of how the reactor has to set things up to get the correct order on a clean build, I can't make the top-level pom a parent of `core/test-api/junit` so it can't inherit properties.